ophyd/__main__.py

```diff
@@ -0,0 +1,16 @@
+from argparse import ArgumentParser
+
+from . import __version__
+
+__all__ = ["main"]
+
+
+def main(args=None):
+    parser = ArgumentParser()
+    parser.add_argument("--version", action="version", version=__version__)
+    args = parser.parse_args(args)
```

Comment:
This is here purely so someone can type `python -m ophyd --version` and get the installed version of ophyd. It is used mainly in CI to test that ophyd imports and gives a version number. It is here because most of my projects sprout a CLI at some point, but I don't know if ophyd ever will, so maybe it is out of place here. If this is too intrusive, I could replace the invocation in CI with `python -c "import ophyd; print(ophyd.__version__)"` to have the same effect.

Reply:
After 1 weekend of brain churning I think its reasonable, I was mostly confused because I hadn't seen an entrypoint sprout up just for versions before. In retrospect with more time to consider it I'm thinking that some of my projects would benefit too, particularly for quickly checking if I'm running some sort of dev version.